### PR TITLE
Updates to MNIST and Model_Quickload Tutorials

### PR DESCRIPTION
**Pull request by inkawhich**
Attn: @orionr @janelu1 

Pull Request for updated MNIST and Model_Quickload tutorials. Changes include formatting, writing, and some code updates. Code updates are mostly in the MNIST tutorial, trying to use Brew as much as possible.

### CODE REVIEW

**Comment by orionr** [comment]
@caffe2bot test this please

File: caffe2/python/tutorials/py_gen/MNIST.py [hunk 1]
```diff
@@ -152,48 +174,61 @@ def AddInput(model, batch_size, db, db_type):
...
 # 
-# In this function we are going to use Brew for the second time. Please refer to the explanation given above. When below we call brew.fc(model, layer, ...) under the hood the following happens. FC operator is going to be added to model.net by calling model.net.FC([layer, W, b], ...). Where W and b are the weight and the bias of this fully connected layer (output = layer * W^T + b). Initially, we get W and b by adding their initialization into model.param_init_net. All of these is happening under the hood. You could just use Brew! :) 
+# In this function we are going to use Brew for the first time. Under the hood, when we call `brew.fc(model, layer, ...)` the following happens:
+# 
+# - The FC operator is added to `model.net` by calling `model.net.FC([layer, W, b], ...)`
+# - The initializations for $W$ and $b$ are added to the `model.param_init_net`
+# 
+# Hopefully, this makes the convenience of brew is clear!
```
Comment (orionr) [inline caffe2/python/tutorials/py_gen/MNIST.py]: Nit: Hopefully, this makes the convenience of brew is clear!  -> Hopefully, this makes the convenience of brew clearer!

File: caffe2/python/tutorials/py_gen/MNIST.py [hunk 2]
```diff
@@ -404,41 +462,69 @@ def AddBookkeepingOperators(model):
...
 pyplot.plot(accuracy, 'r')
+pyplot.title("Summary of Training Run")
+pyplot.xlabel("Iteration")
 pyplot.legend(('Loss', 'Accuracy'), loc='upper right')
 
 
-# Now we can sample some of the data and predictions. 
+# ## Visualize Results
+# 
+# Now that our model is trained we can visualize the results of our labors. But first, check out what was printed during the training run above. In the first iteration, the training accuracy for batch 0 was 3.1%, but it quickly converged to mid-nineties by the 100th iteration. The plot is also shows this trend.
```
Comment (orionr) [inline caffe2/python/tutorials/py_gen/MNIST.py]: Nit: The plot is also shows this trend. -> The plot also shows this trend.

File: caffe2/python/tutorials/py_gen/MNIST.py [hunk 3]
```diff
@@ -404,41 +462,69 @@ def AddBookkeepingOperators(model):
...
-# Let's look at some of the data.
+### Let's look at some of the training data
 pyplot.figure()
+pyplot.title("Training Data Sample")
+# Grab the most recent data blob (i.e. batch) from the workspace
 data = workspace.FetchBlob('data')
+# Use visualize module to show the examples from the last batch that was fed to the model
 _ = visualize.NCHW.ShowMultiple(data)
+
+### Lets visualize the softmax result
```
Comment (orionr) [inline caffe2/python/tutorials/py_gen/MNIST.py]: Nit: Lets -> Let's

File: caffe2/python/tutorials/py_gen/Model_Quickload.py [hunk 4]
```diff
@@ -10,39 +10,52 @@
 
 # # Model Quickload
 # 
-# This short notebook will show you how you can very quickly load and test SqueezeNet, which is a very small and fast model based on AlexNet and is useful for identifying objects. The range of objects groups is only 1,000.
+# This notebook will show you how you can very quickly load a pretrained SqueezeNet model and test it on images of your choice in four main steps.
```
Comment (orionr) [inline caffe2/python/tutorials/py_gen/Model_Quickload.py]: Nit: This notebook will show you how you can very quickly -> This notebook will show you how to quickly

**Review by orionr — APPROVED** [review]
Nicely done! Is this the tutorial where you were having convergence issues? If so, let's figure that out before landing, but this looks great.

**Comment by inkawhich** [comment]
@caffe2bot test this please

**Comment by inkawhich** [comment]
@caffe2bot retest this please

**Comment by orionr** [comment]
@caffe2bot retest this please

**Comment by orionr** [comment]
Looks good @inkawhich! Once all tests pass feel free to squash and merge.

**Pull request merged**